### PR TITLE
Update migrations batch file + Review migration diffs

### PR DESCRIPTION
**Pull request by HaoK**
Part of https://github.com/aspnet/templating/issues/353

@ajcvickers so the automated updateMigrations.cmd is producing slightly different migrations than what we have checkedin, this PR is to review both the script (is there a way to specify the number in front of the schema?  Ideally I want it to use 000000000000 like we have hardcoded).

And the diffs... which appear to mostly be ordering in the designer

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs [hunk 1]
```diff
@@ -1,20 +1,65 @@
...
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
 using Microsoft.EntityFrameworkCore.Migrations;
 
 namespace Company.WebApplication1.Data.Migrations
 {
     [DbContext(typeof(ApplicationDbContext))]
-    [Migration("00000000000000_CreateIdentitySchema")]
+    [Migration("20180419233801_CreateIdentitySchema")]
```
Comment (HaoK) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs]: Is there anyway to hardcode this to 00000000000000_CreateIdentitySchema?

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs [hunk 2]
```diff
@@ -1,20 +1,65 @@
...
     partial class CreateIdentitySchema
     {
         protected override void BuildTargetModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.1.0-preview1");
+                .HasAnnotation("ProductVersion", "2.1.0-preview2-30571");
```
Comment (HaoK) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs]: We still need to close on what version string we want here, I guess we can just hand update this to 2.1.0-final/rtm or whatever we want as well

**Comment by HaoK** [comment]
This makes the updates a little bit better, so as things currently stand, the two remaining warts are in the designer:
```
    [Migration("20180419233811_CreateIdentitySchema")]

                .HasAnnotation("ProductVersion", "2.1.0-preview2-30571")
```

Even if we have to hand edit those by hand x4, I guess its not too bad.

**Comment by ajcvickers** [comment]
@bricelam A few things here:
* Looks like we're including an internal namespace in the snapshot now--is this expected?
* There's a whitespace change that doesn't look right
* Can you answer @HaoK's question on the timestamp?

@HaoK This is not for RC1, correct?

**Comment by HaoK** [comment]
@ajcvickers no this is just prep work for 2.1 RTM or whatever the next milestone after rc1 is

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs (hunk 1 above)
Comment (bricelam) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs]: Not from command-line. You can override the `IMigrationsIdGenerator` service...

**Comment by bricelam** [comment]
It would be good to understand where the `.Internal` is coming from. AFAIK, we should be generating a lot less of these--not more.

The witespace change looks correct in the new code. Was the old one a manual-edit mistake?

**Comment by HaoK** [comment]
@bricelam probably I think its been there a long time

**Comment by bricelam** [comment]
Did you use 2.1.0-preview2 to generate? I suspect 2.1.0-rc1 will not generate `using ...Internal`

**Comment by HaoK** [comment]
Its using whatever's in dev's templating repo right now, I can double check to see what version it is

**Comment by HaoK** [comment]
Replaced by https://github.com/aspnet/templating/pull/484

**Comment by ajcvickers** [comment]
@HaoK Didn't mean to approve this one--I have both tabs open and approved the wrong tab. 🐑

**Pull request closed without merging**